packages/pn-pa-webapp/src/pages/components/Notifications/FilterNotificationsFormBody.tsx

```diff
@@ -66,12 +66,9 @@ const FilterNotificationsFormBody = ({
 
   const handleChangeTouched = async (e: ChangeEvent) => {
     if (e.target.id === 'iunMatch') {
-      const newInput = formatIun(formikInstance.values.iunMatch, (e.nativeEvent as any).data);
-      if (newInput) {
-        await formikInstance.setFieldValue('iunMatch', newInput);
-      } else {
-        formikInstance.handleChange(e);
-      }
+      const upperCaseValue = (e.target as HTMLInputElement).value.toUpperCase();
```

Comment:
Non dovrebbe essere necessario fare due volte l'uppercase.
In realtà se si ritorna al codice precedente, basta fare l'uppercase sulla variabile newInput